packages/home-service/.env.example

```
@@ -15,3 +15,10 @@ USER_SERVICE_SERVICE_PORT=
 ## Redis Host
 REDIS_SERVICE_HOST=redis
 # REDIS_SERVICE_PORT=
+
+## API Gateway
+API_GATEWAY=
+GATEWAY_AUTH_TOKEN=
+
+## URLs
+CLIENT_URL=
```

```suggestion
CLIENT_URL=

```

Newline at the end of the file.